README.md

```diff
@@ -20,6 +20,12 @@ It currently also experimentally supports the injection of
 a backward compatible way with NGSI-v2 API. I.e. you can retrieve NGSI-LD stored data via NGSI v2
 API and retrieve data will be describe following NGSI v2 format.
 
+QuantumLeap is not a [real time](https://en.wikipedia.org/wiki/Real-time_computing) API,
+its purpose is to process notifications received from the Context Broker and
+to create temporal records for them. In general, the whole FIWARE stack, being
+based on a micro-service architecture, cannot be regarded as real time
+in case you have requirements on guaranteed delivery in a given amount of time.
+
```

Comment:
I'd also add a para along the lines of the following, just to make sure people out there don't interpret our clarification as advice not to use FiWare because of poor performance:

> However, even though hard real time may not be FIWARE's forte, in our experience a properly tuned FIWARE stack can perform extremely well and handle very demanding IoT workloads without a glitch.

I've also [added a comment](https://github.com/smartsdk/ngsi-timeseries-api/issues/418#issuecomment-764556047) to #418 along these lines just to make sure it's crystal clear what we're talking about here is a corner case (hard real time) whereas in the majority of cases FIWARE provides all the performance you'll ever need :-)